draft-ietf-rats-eat.md

```diff
@@ -972,8 +972,9 @@ For example, it may be desirable to report the results for measurements of the f
 Note that this claim is not for reporting the overall result of a verifier.
 It is solely for reporting the result of comparison to reference values.
 
-An individual measurement result is an array of two, an identifier of the measurement and an enumerated type that is the result.
-The range and values of the measurement identifier varies from one measurement scheme to another.
+An individual measurement result (individual-result) is an array of two, an identifier of the measurement (result-id) and an enumerated type that is the result (result).
+Different measurement systems will measure different things and perhaps measure the same thing in different ways.
```

Comment:
```suggestion
Different measurement systems may measure different Target Environments, or perhaps measure the same Target Environment differently.
```

Reply:
While the target of an attestation is a target environment, the target of a measurement system here is not necessary a target environment. This is for use in many other contexts than measured boot or measurements in the TCG sense.